halo2-ecc/src/secp256k1/tests/schnorr_signature.rs

```diff
@@ -68,7 +67,7 @@ pub fn random_schnorr_signature_input(rng: &mut StdRng) -> SchnorrInput {
     let mut y: &Fp = r_point.y();
     // make sure R.y is even
     while fe_to_biguint(y).mod_floor(&BigUint::from(2u64)) != BigUint::from(0u64) {
-        k = <Secp256k1Affine as CurveAffine>::ScalarExt::random(rng.clone());
+        k = <Secp256k1Affine as CurveAffine>::ScalarExt::random(StdRng::from_seed([0u8; 32]));
```

Comment:
`rng` (otherwise `k` is the same on every run of this function)

Reply:
yes, exactly, sry i was seriously concerned by some stuff and not in good state/clear mind when writing this, this is a stupid mistake..

Reply:
yes, exactly, sry i was seriously concerned/worried by some stuff and not in good state/clear mind when writing this, this is a stupid mistake..